expr/func_keeplastvalue.go

```diff
@@ -28,6 +28,7 @@ func (s *FuncKeepLastValue) Signature() ([]Arg, []Arg) {
 					// Treats any string as infinity. This matches Graphite's behavior
 					// (although intended bevahior is to let user specify "INF" as the limit)
 					ArgString{val: &stub},
+					ArgQuotelessString{val: &stub},
```

Comment:
are we declaring here that we need a string argument followed by a quotelessstring argument?
this doesn't seem to make sense. should we use `ArgIn` (which is what you'd use if you want to allow one of more types of args)

Reply:
d'oh, i didn't expand the diff enough. this is precisely what you're doing...